docs/examples/networking/nat/env-vars

```diff
@@ -0,0 +1,8 @@
+### Region
+export TF_VAR_region=""
+
+### Compartment
+export TF_VAR_compartment_ocid=""
+
+export TF_VAR_ssh_public_key=$(cat .../id_rsa.pub)
```

Comment:
How about the follow, copied from another example:

export TF_VAR_ssh_public_key=$(cat <path to public key>)
export TF_VAR_ssh_private_key=$(cat <path to private key>)